snomed/com.b2international.snowowl.validation.snomed/src/test/java/com/b2international/snowowl/validation/snomed/GenericValidationRuleTest.java

```diff
@@ -499,5 +499,36 @@ public void rule670() throws Exception {
 		assertAffectedComponents(issues, 
 				ComponentIdentifier.of(SnomedTerminologyComponentConstants.CONCEPT_NUMBER, Concepts.IS_A));
 	}
+
+	@Test
+	public void rule671() throws Exception {
+		final String ruleId = "671";
+		indexRule(ruleId);
+		
+		String conceptId = generateConceptId();
+		SnomedDescriptionIndexEntry fsn = description(generateDescriptionId(), Concepts.FULLY_SPECIFIED_NAME, "Fully specified name 3 (tag)")
+				.conceptId(conceptId)
+				.acceptability(Concepts.REFSET_LANGUAGE_TYPE_ES, Acceptability.PREFERRED)
+				.build();
+		SnomedDescriptionIndexEntry pt = description(generateDescriptionId(), Concepts.SYNONYM, "Preferred term 3")
+				.acceptability(Concepts.REFSET_LANGUAGE_TYPE_ES, Acceptability.PREFERRED)
+				.conceptId(conceptId)
+				.build();
+		SnomedRefSetMemberIndexEntry ptMember = member(pt.getId(), SnomedTerminologyComponentConstants.DESCRIPTION_NUMBER, Concepts.REFSET_DESCRIPTION_INACTIVITY_INDICATOR).valueId(Concepts.CONCEPT_NON_CURRENT).build();
```

Comment:
There is no test for active description with incorrect indicator value in the refset.
There is no test for inactive description without any indicator value in the refset.

Reply:
Done